membership-attribute-service/app/controllers/HealthCheckController.scala

```diff
@@ -32,7 +44,7 @@ class HealthCheckController extends Results {
       if (failures.isEmpty) {
         Ok(Json.obj("status" -> "ok", "gitCommitId" -> app.BuildInfo.gitCommitId))
       } else {
-        failures.flatMap(_.messages).foreach(msg => Logger.warn(msg))
+        failures.flatMap(_.messages).foreach(msg => logger.error(msg))
```

Comment:
I am guessing that this was at warn level to stop us picking up errors in Sentry on every new deployment (this is just a theory though)

Reply:
Ah yes, ok I'll change that.